proto/evm/v1beta1/genesis.proto

```diff
@@ -4,10 +4,28 @@ option go_package = "github.com/axelarnetwork/axelar-core/x/evm/types";
 
 import "gogoproto/gogo.proto";
 import "evm/v1beta1/params.proto";
+import "evm/v1beta1/types.proto";
 
 option (gogoproto.goproto_getters_all) = false;
 
 // GenesisState represents the genesis state
 message GenesisState {
-  repeated Params params = 1 [ (gogoproto.nullable) = false ];
-}
\ No newline at end of file
+  option (gogoproto.stable_marshaler) = true;
+
+  message Chain {
+    Params params = 1 [ (gogoproto.nullable) = false ];
```

Comment:
Can the `gov` module still vote to update params in here?

Reply:
the way to store has not changed, they are still stored in paramskeeper subspaces. So if the gov module was able to vote on them before it's still possible